client/layout/guided-tours/config-elements.js

```diff
@@ -419,7 +422,25 @@ export class Continue extends Component {
 			return null;
 		}
 
-		return <i>{ this.props.children || translate( 'Click to continue.' ) }</i>;
+		return (
+			<p className="guided-tours__actionstep-instructions">
+				<i>{ this.props.children || translate( 'Click to continue.' ) }</i>
+			</p>
+		);
+	}
+}
+
+export class ButtonRow extends Component {
```

Comment:
Not at all mandatory, but such a simple and lightweight component could be downgraded to a pure function.

```js
export const ButtonRow = ( { children, single } ) => (
  <div …
```